dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -88,6 +97,44 @@ private static void generateReports(Configuration configuration, Path output) {
     }
   }
 
+  @VisibleForTesting
+  static List<String> readBom() throws ArtifactDescriptorException {
+    
+    DefaultArtifact artifact =
+        new DefaultArtifact("com.google.cloud:cloud-oss-bom:pom:0.62.0-SNAPSHOT");
+
+    RepositorySystem system = RepositoryUtility.newRepositorySystem();
+    RepositorySystemSession session = RepositoryUtility.newSession(system);
+
+    ArtifactDescriptorRequest request = new ArtifactDescriptorRequest();
+    request.addRepository(RepositoryUtility.CENTRAL);
+    request.setArtifact(artifact);
+
+    ArtifactDescriptorResult resolved = system.readArtifactDescriptor(session, request);
+    List<Exception> exceptions = resolved.getExceptions();
+    if (!exceptions.isEmpty()) {
+      throw new ArtifactDescriptorException(resolved, exceptions.get(0).getMessage());
+    }
+    
+    List<String> result = new ArrayList<>();
+    for (Dependency dependency : resolved.getManagedDependencies()) {
+      Artifact managed = dependency.getArtifact();
+      // TODO remove this hack once we get these out of 
+      // google-cloud-java's BOM
+      if (managed.getArtifactId().contains("logback")
+          || managed.getArtifactId().contains("contrib")) {
+        continue;
+      }
+      String coords =
+          managed.getGroupId() + ":" + managed.getArtifactId() + ":" + managed.getVersion();
```

Comment:
Don't we have `Artifacts.toCoordinates(Artifact)` for this?

Reply:
done